### ISSUE TITLE
"Request textDocument/hover failed" in Vagrantfile

### ISSUE THREAD

**Issue by fomentia** [castwide/vscode-solargraph#40]
I'm going through the [Consul "Getting Started" tutorial](https://www.consul.io/intro/getting-started/join.html) in my free time. I downloaded their [demo Vagrantfile](https://github.com/hashicorp/consul/blob/master/demo/vagrant-cluster/Vagrantfile) and opened it in Visual Studio Code. The global `$script` had a warning (I'm sure you can guess what for). When I hovered over it, the output pane popped up with this error message:

```
[Error - 07:57:30] Request textDocument/hover failed.
  Message: undefined method `empty?' for nil:NilClass
  Code: -32603
```

The output pane target was set to the Ruby Language Server.

I _might_ take a look at fixing it myself (no promises), but I figured I ought to at least make the issue known. :)

**Comment by castwide** [comment]
Thanks. Coincidentally, this bug is fixed in gem version 0.18.3, which was released this morning.

**Comment by castwide** [comment]
Thanks. Coincidentally, this bug is fixed in gem version 0.18.3., which was released this morning.

**Comment by fomentia** [comment]
@castwide Indeed! After an update, the problem disappeared. Loving the project; keep up the good work! 👍
